### ISSUE TITLE
Support figcaption?

### ISSUE THREAD

**Issue by chucker** [trenz-gmbh/trenz-docs#22]
Markdig has [syntax for figure captions](https://github.com/xoofx/markdig/blob/master/src/Markdig.Tests/Specs/FigureFooterAndCiteSpecs.md#figures).

However, Azure DevOps doesn't seem to currently support this either.

**Comment by ricardoboss** [comment]
We can already use HTML code within markdown since Marked.js supports this already: https://github.github.com/gfm/#html-blocks

**Comment by ricardoboss** [comment]
Tested using https://github.com/ricardoboss/trenz-docs-test/blob/937dc5d12f2b64a6440aa09cb61be2cd6f253f11/docs/GHFM.md?plain=1#L41

**Comment by chucker** [comment]
> This brought a new issue to light: the link to the image within the `figure` doesn't get replaced...

Yes, this wouldn't work. Both Azure DevOps and TRENZ Docs get confused by the relative URL.
